config/research/techwebs_config.txt

```diff
@@ -0,0 +1,22 @@
+##While this may look like a regular config file, it is not parsed the same as a regular config file. # still denotes comments.
```

Comment:
Considering we have a whole system to configs to make them modular, why isn't this parsed using the build-in config system?

@Cyberboss set this man straight

Reply:
@MrStonedOne the regular configs require individual config entry datums for something to work, the techwebs primary one is just a list of files to include, the other files are processed by the subsystem. 2/4 of the types of files will just be .jsons, one of them is basically al ist of jsons by what the json is supposed to do, and the fourth is this one.